common/ipfs_util.py

```diff
@@ -2,13 +2,26 @@
 import json
 import logging
 
-import ipfsapi
+import ipfshttpclient
 
 
 class IPFSUtil(object):
 
     def __init__(self, ipfs_url, port):
-        self.ipfs_conn = ipfsapi.connect(host=ipfs_url, port=port)
+        #
+        # Please refer to the documentation given at  https://libraries.io/pypi/ipfsapi
+        # The API deamon location is now described using MultiAddr,
+        # hence rather than doing ipfshttpclient.connect(host, port) to pass the network address parameters,
+        # use:ipfshttpclient.connect("/dns/<host>/tcp/<port>/http"),
+        # Please note that https://ipfs.singularitynet.io/api/v0/version
+        # {
+        # "Version": "0.4.19",
+        # "Commit": "",
+        # "Repo": "7",
+        # "System": "amd64/linux",
+        # "Golang": "go1.11.5"
+        # }
+        self.ipfs_conn = ipfshttpclient.connect(f'/dns/{ipfs_url}/tcp/{port}/https', session=False)
```

Comment:
Do we need to customize the ipfs_url in the config file?

Reply:
no we just set it as  , i**pfs.singularitynet.io**, in config, 
 test case confirms this value , you need to pass it as using [[MultiAddr](https://github.com/multiformats/multiaddr)],